src/plugins/timeConductor/ConductorHistory.vue

```diff
@@ -39,6 +39,9 @@ const DEFAULT_DURATION_FORMATTER = 'duration';
 const LOCAL_STORAGE_HISTORY_KEY_FIXED = 'tcHistory';
 const LOCAL_STORAGE_HISTORY_KEY_REALTIME = 'tcHistoryRealtime';
 const DEFAULT_RECORDS = 10;
+const ONE_MINUTE = 60 * 1000;
+const ONE_HOUR = ONE_MINUTE * 60;
+const TWENTYFOUR_HOURS = ONE_HOUR * 24;
```

Comment:
TWENTY_FOUR_HOURS 😃

Reply:
haha, good point, you know, I think I saw that, then didn't do anything about it...